### PR TITLE
permitAndStake documentation

### LINKED ISSUE #78: Docs change 2
https://github.com/cantinasec/review-uniswapfoundation/issues/5

### PR DESCRIPTION
**Pull request by wildmolasses**
Resolves #78

### CODE REVIEW

File: src/UniStaker.sol [hunk 1]
```diff
@@ -609,7 +607,8 @@ contract UniStaker is INotifiableRewardReceiver, Multicall, EIP712, Nonces {
   function notifyRewardAmount(uint256 _amount) external {
     if (!isRewardNotifier[msg.sender]) revert UniStaker__Unauthorized("not notifier", msg.sender);
 
-    // We checkpoint the accumulator without updating the timestamp at which it was updated, because
+    // We checkpoint the accumulator without updating the timestamp at which it was updated,
+    // because
```
Comment (alexkeating) [inline src/UniStaker.sol:611]: Weird wrapping here

**Review by alexkeating — APPROVED** [review]
Looks good as long as that weird wrapping is resolved

File: src/UniStaker.sol (hunk 1 above)
Comment (apbendi) [reply src/UniStaker.sol:611]: Huh yeah, why did this line get changed at all? Otherwise change looks good 👍

**Review by apbendi — APPROVED** [review]

File: src/UniStaker.sol (hunk 1 above)
Comment (wildmolasses) [reply src/UniStaker.sol]: thanks for the catch, fixed in c4d3e93

**Pull request merged**